Configuration simulated devices updated

sandervandevelde opened the pull request:
Conform https://github.com/Azure/iot-edge/blob/master/modules/simulated_device/devdoc/simulated_device_requirements.md an integer message period must be added. Arguments for both simulated devices are updated.

azurecla commented:
Hi __@sphera5__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

cjgronlund commented:
@sphera5 
Thank you for your contribution to Azure documentation. I've sent your recommendation for review.

Best regards,
Carolyn
Azure documentation

cjgronlund commented:
@sphera5 
Thanks again for your contribution. The product team prefers to leave the choice of Java SDK up to the user. But, the feedback is useful for future consideration.

Regards,

Carolyn
Azure Documentation Team

sandervandevelde committed:
Configuration simulated devices updated

Conform https://github.com/Azure/iot-edge/blob/master/modules/simulated_device/devdoc/simulated_device_requirements.md an integer message period must be added. Arguments for both simulated devices are updated.
